lib/components/Link/Link.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import classnames from 'classnames';
+import {
+  useLinkComponent,
+  LinkComponentProps,
+} from '../BraidProvider/BraidProvider';
+import * as resetStyles from '../../reset';
+
+export type LinkProps = LinkComponentProps;
+
+export const Link = ({ href, className, ...restProps }: LinkComponentProps) => {
+  const LinkComponent = useLinkComponent();
+
+  return (
+    <LinkComponent
+      href={href}
+      className={classnames(resetStyles.base, resetStyles.element.a, className)}
```

Comment:
ah gotcha, just saw this.

any reason not to use `useBoxStyles({ component: 'a', className })`?

Reply:
Yeah, that probably makes more sense 👍